contracts/test/validator/TestValidatorSet.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "../../validator/ValidatorSet.sol";
+
+/**
+ * @title TestValidatorSet contract.
+ *
+ * @notice It is used for testing ValidatorSet contract
+ */
+contract TestValidatorSet is ValidatorSet {
+
+    /* External functions */
+
+    /**
+     * @notice It is used to insert validator.
+     *
+     * @param _validator Validator address.
+     * @param _beginHeight Begin height for the validator.
+     */
+    function addValidator(address _validator, uint256 _beginHeight) external {
```

Comment:
Why do we need to change names? If it's only because insertValidator is already there in base class, let's mark it as internal in base class: `ValidatorSet::insertValidatorInternal`, thoughts?